src/main/java/me/partlysanestudios/partlysaneskies/OneConfigScreen.java

```diff
@@ -418,6 +418,14 @@ public void resetBrokenStrings() {
     )
     public boolean getDataOnJoin = true;
 
+    @Switch(
```

Comment:
Alright this is nice! Could you add the other constant value (other comment) to the config, You should be able to just create one number config element